coss/opensource_clubs/templates/opensource_clubs/entity_detail_page.html

```diff
@@ -120,23 +120,13 @@ <h3 class="highlight mt-4">{{ page.title }}</h3>
         {% include 'home/blocks/image_gallery.html' with gallery=page.gallery_images %}
       </div>
     </div>
-
-    <div class="row mt-5 mb-5">
-      <div class="col-md-6">
-        <h6>
-          SHARE THIS CLUB ON SOCIAL:
-          <a href="https://www.facebook.com/sharer/sharer.php?u={{ page.full_url }}">
-            <img src="{% static 'img/icons/facebook-black-large.svg' %}"></img>
-          </a>
-          <a href="https://www.twitter.com/share?url={{ page.full_url }}&text={{ page.title }}">
-            <img src="{% static 'img/icons/twitter-black.svg' %}"></img>
-          </a>
-        </h6>
-      </div>
-    </div>
   </div>
 {% endblock %}
 
+{% block bottom_content %}
```

Comment:
The social component is not matching the design. 
The yellow line is missing and the icons are too close to each other. Moreover we are missing the report this club link.
![screenshot-2017-9-14 club1](https://user-images.githubusercontent.com/900565/30423882-e9b58ba6-994c-11e7-8159-f9bfcdb50a25.png)